docs/http/http-spans.md

```diff
@@ -61,13 +61,20 @@ and various HTTP versions like 1.1, 2 and SPDY.
 ## Name
 
 HTTP spans MUST follow the overall [guidelines for span names](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.22.0/specification/trace/api.md#span).
-HTTP server span names SHOULD be `{http.request.method} {http.route}` if there is a
-(low-cardinality) `http.route` available.
-HTTP server span names SHOULD be `{http.request.method}` if there is no (low-cardinality)
-`http.route` available.
+
+HTTP server span names SHOULD be `{method} {http.route}` if there is a
```

Comment:
It is unclear what "method" is at this point, since it's only defined later.

```suggestion
HTTP server span names SHOULD be `{method} {http.route}` (see below for the exact definition of the method placeholder) if there is a
```